frontend/src/css/SubmitAvailability.css

```
@@ -0,0 +1,480 @@
+    width: 100%;
+    padding: 0.625rem 2.5rem 0.625rem 0.875rem;
+  }
+}
+
+/* Teacher Select */
+.teacher-select {
+  width: 100%;
+  max-width: 320px;
+  padding: 0.875rem 1.25rem;
+  font-size: 0.9rem;
+  font-family: 'Inter', sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)